packages/ui/src/helpers/authenticator/formFields/formFields.ts

```diff
@@ -31,6 +31,7 @@ export const getFormFields = (
   const customFormFields =
     getActorState(state).context?.formFields?.[route] || {};
   const formFields = applyDefaults(defaultFormFields, customFormFields);
+  delete formFields['QR'];
```

Comment:
QR isn't a field, so it doesn't need to be in this list. It's only used in the Setup TOTP page, to check for the `totpIssuer` and `totpUsername`. Which this should not affect.

Reply:
In that case seems like we should prevent the `QR` key from being added to `formFields` altogether?

Reply:
That's a good point. 

The formFields is passed in from the customer. To change it, it would break. https://ui.docs.amplify.aws/components/authenticator#update-setup-totp-qr-issuer-and-username

But maybe something we can add in 3.0?

We could try to delete the QR code earlier on, in the `getSortedFormField`, but I hesitated doing it there, because it would require me to do a deep clone, and that's already done in the `applyDefaults`. Otherwise when you delete it, it deletes it everywhere.

Reply:
Updating for the 3.0 release makes sense 👍🏽 Do you think it make sense to remove the `QR` key/values from `formFields` as part of the breaking change? I might be misunderstanding the docs, but based on the example it looks like the `QR` field is not actually a form field.

Reply:
Yeah, I'm thinking maybe we can do that. That would require us to create yet another prop that the Authenticator would accept...